docs: Adds a backup step to the DragonFish migration guide (#684)
truecharts/website: src/content/docs/SCALE/migrations/cobia-dragonfish.mdx

## For Existing Users and Apps

### Pre Cobia-to-DragonFish Upgrade Steps

<Steps>

1. Ensure apps are up-to-date on Cobia

>     Prior to updating to TrueNAS SCALE DragonFish, please ensure that TrueNAS SCALE is updated to the latest version of Cobia (23.10.2), and that all of your TrueCharts apps are fully updated, on the correct trains and running or otherwise working correctly.

Reviewer: ```suggestion
    Prior to updating to TrueNAS SCALE DragonFish, please ensure that TrueNAS SCALE is updated to the latest version of Cobia (23.10.2), and that all of your TrueCharts apps are fully updated, on the [correct trains]((/news/train-renames) and running or otherwise working correctly.
```

2. Create a backup using HeavyScript

    For details on installing and creating a backup with HeavyScript, please see the [HeavyScript GitHub Repo.](https://github.com/Heavybullets8/heavy_script)

</Steps>

:::note[Upgrade to Dragonfish]
    

>     Update as usual through TrueNAS' System Settings > Update page.

Reviewer: ```suggestion
    Update to DragonFish as usual through TrueNAS' System Settings > Update page.
```

Sometimes, things don't go as smoothly as they should. These are tips to help with any known issues identified as part of the migration process.

### CNPG database cannot be restored via the migration script

For instances where an application's database cannot be restored, the script will prompt you to select a backup (created with HeavyScript) instead. This guide assumes HeavyScript has been installed under the current user's home directory.

1. Navigate to HeavyScript's backup folder for the selected application and list database dumps.

    ```shell title="Check for Backups"
    cd ~/heavy_script/database_dumps/APPNAME && ls
    ```

2. Copy the latest database dump to the migration dataset created by the migration script.

    ```shell title="Copy Database"
    cp APPNAME_TIMESTAMP.sql.gz /mnt/TANK/migration/APPNAME/backup/
    ```

> 3. Rerun the migration process and select the backup.

Reviewer: ```suggestion
3. Re-run the migration process and select the backup.
```
